src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -62,9 +62,9 @@ object MethodRepository {
     val ListAll = List("All")
   }
 
-  def validateEmail(email:String): Boolean = {
+  def validatePublicOrEmail(email:String): Boolean = {
     // TODO: real email address validation!
-    !email.trim.isEmpty
+    !email.trim.isEmpty || email.trim.equalsIgnoreCase("public")
```

Comment:
per previous comment ... leave off the second part of this.

if `!email.trim.isEmpty` is false, `email.trim.equalsIgnoreCase("public")` can never be true, and it's just extra work for the jvm to calculate.